### PR TITLE
xournalpp: Add version 1.0.15

### PR DESCRIPTION
**Pull request by simnalamburt**
Xournal++ is a handwriting notetaking software with PDF annotation support. Supports pen input from devices such as Wacom Tablets.

Xournal++ sample screenshot

NOTE: There's [an issue](https://github.com/xournalpp/xournalpp/issues/1600) with Xournal++ 1.0.16 so I'm hereby make a PR with version 1.0.15

### CODE REVIEW

File: bucket/xournalpp.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.15",
+    "description": "A handwriting notetaking software with PDF annotation support",
+    "homepage": "https://github.com/xournalpp/xournalpp",
+    "license": "GPL-2.0",
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
    "license": "GPL-2.0-only",
```

File: bucket/xournalpp.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        "script": "Expand-7zipArchive \"$dir\\xournalpp-$version-windows.exe\""
+    },
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\" -Recurse",
+    "shortcuts": [
+        [
+            "bin\\xournalpp.exe",
+            "Xournal++"
+        ]
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
    "checkver": "github",
```

File: bucket/xournalpp.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.15",
+    "description": "A handwriting notetaking software with PDF annotation support",
+    "homepage": "https://github.com/xournalpp/xournalpp",
+    "license": "GPL-2.0",
+    "url": "https://github.com/xournalpp/xournalpp/releases/download/1.0.15/xournalpp-1.0.15-windows.zip",
+    "hash": [
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
            "hash": "42b0dc65992ea51938aa9fde2d874e341f4005995fde6ab1d33ae117beb2ea5d"
        }
    },
```

File: bucket/xournalpp.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "sha256:42b0dc65992ea51938aa9fde2d874e341f4005995fde6ab1d33ae117beb2ea5d",
+        "sha512:5a282ecff9d231f4bd0d0754e17258a9bb90f12e7941c9cafea3bf42f82294f061ef1fae1c5092c66472e121555832c576bfea4a8fc5d05d73b6deda6096e576"
+    ],
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\xournalpp-$version-windows.exe\""
+    },
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\" -Recurse",
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
    "bin": "bin\\xournalpp.exe",
    "shortcuts": [
```

File: bucket/xournalpp.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        [
+            "bin\\xournalpp.exe",
+            "Xournal++"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/xournalpp/xournalpp"
+    },
+    "autoupdate": {
+        "url": "https://github.com/xournalpp/xournalpp/releases/download/$version/xournalpp-$version-windows.zip"
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/xournalpp/xournalpp/releases/download/$version/xournalpp-$version-windows.zip"
            }
        }
```

File: bucket/xournalpp.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.15",
+    "description": "A handwriting notetaking software with PDF annotation support",
+    "homepage": "https://github.com/xournalpp/xournalpp",
+    "license": "GPL-2.0",
+    "url": "https://github.com/xournalpp/xournalpp/releases/download/1.0.15/xournalpp-1.0.15-windows.zip",
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/xournalpp/xournalpp/releases/download/1.0.15/xournalpp-1.0.15-windows.zip",
```

File: bucket/xournalpp.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    "hash": [
+        "sha256:42b0dc65992ea51938aa9fde2d874e341f4005995fde6ab1d33ae117beb2ea5d",
+        "sha512:5a282ecff9d231f4bd0d0754e17258a9bb90f12e7941c9cafea3bf42f82294f061ef1fae1c5092c66472e121555832c576bfea4a8fc5d05d73b6deda6096e576"
+    ],
+    "installer": {
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
    "installer": {
        "script": [
            "Expand-7zipArchive \"$dir\\xournalpp-$version-windows.exe\" -Removal",
            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\" -Recurse"
        ]
    },
```

File: bucket/xournalpp.json (hunk 1 above)
Comment (simnalamburt) [reply bucket/xournalpp.json]: Actually it's GPL-2.0-or-later. I'll update it as "GPL-2.0-or-later"

File: bucket/xournalpp.json [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    "version": "1.0.15",
+    "description": "A handwriting notetaking software with PDF annotation support",
+    "homepage": "https://github.com/xournalpp/xournalpp",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit" : {
+            "url": "https://github.com/xournalpp/xournalpp/releases/download/1.0.15/xournalpp-1.0.15-windows.zip"
+        }
+    },
+    "hash": "42b0dc65992ea51938aa9fde2d874e341f4005995fde6ab1d33ae117beb2ea5d",
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
```

File: bucket/xournalpp.json [hunk 9]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.15",
+    "description": "A handwriting notetaking software with PDF annotation support",
+    "homepage": "https://github.com/xournalpp/xournalpp",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit" : {
+            "url": "https://github.com/xournalpp/xournalpp/releases/download/1.0.15/xournalpp-1.0.15-windows.zip"
```
Comment (Ash258) [inline bucket/xournalpp.json]:
```suggestion
            "url": "https://github.com/xournalpp/xournalpp/releases/download/1.0.15/xournalpp-1.0.15-windows.zip",
            "hash": "42b0dc65992ea51938aa9fde2d874e341f4005995fde6ab1d33ae117beb2ea5d"
```

**Comment by Ash258** [comment]
Why did you remove autoupdate? Checkver is enough.

**Comment by simnalamburt** [comment]
The issue https://github.com/xournalpp/xournalpp/issues/1600 hasn't fixed for quite long time, so let's go with plan "remove checkver and add it again later".

> Why did you remove autoupdate? Checkver is enough.

I wasn't sure about the behavior of scoop, I'll fix it.

**Pull request merged**